test/unit/rules/attributes-order-test.js

```diff
@@ -0,0 +1,310 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+const invalidConfigMessage = 'Your config does not match the allowed values';
```

Comment:
Can you export this error message from the rule? Other rules do this with `ERROR_MESSAGE`.